src/main/app/forms/models/multiRowForm.ts

```diff
@@ -0,0 +1,43 @@
+import { Serializable } from 'models/serializable'
+import { ItemInMultiRowForm } from 'forms/models/itemInMultiRowForm'
+
+export abstract class MultiRowForm<T extends ItemInMultiRowForm> implements Serializable<MultiRowForm<T>> {
+
+  rows: T[]
```

Comment:
If we're making an abstraction, why require the rows to be stored in any particular way? Maybe a getter method returning a ReadonlyArray would be more appropriate?